tests/unit/spec/base/core-datalayer.js

```diff
@@ -10,15 +10,14 @@ describe('core-datalayer.js', function() {
 
     describe('pageHasDownload', function() {
 
-        afterEach(function() {
-            $('.download').remove();
-        });
-
```

Comment:
Should the `$('.download').remove();` be replaced with some sort of `...parentNode.removeChild(content)` call?

Reply:
Only the first test in this block modifies the DOM, so I moved the cleanup code to the test in question, else this will throw an error when the second test tries to remove an element that's not there.